code/go/site/static/code_snippets/getting_started/multiple_events.csharpsnippet

```diff
@@ -0,0 +1,4 @@
+sse.MergeFragments("<div id="question">...</div>");
```

Comment:
Is this valid? Shouldn’t the double quotes be escaped?

Reply:
You are correct. Missed it somehow.